samples/react-list-search/.yo-rc.json

```diff
@@ -1,12 +1,17 @@
 {
   "@microsoft/generator-sharepoint": {
-    "version": "1.13.1",
+    "version": "1.20.0",
     "libraryName": "list-search-webpart",
     "libraryId": "8277f088-9c30-4f95-9c15-9c18a9d40a26",
     "environment": "spo",
     "packageManager": "npm",
     "isCreatingSolution": false,
     "isDomainIsolated": false,
-    "componentType": "webpart"
+    "componentType": "webpart",
+    "nodeVersion": "20.17.0",
```

Comment:
for SPFx (at least now) it is recommended not to go above v18. 
Lets include a recommended path in the sample 👍